devicetypes/smartthings/vision-in-wall-2relays-switch.src/vision-in-wall-2relays-switch.groovy

```diff
@@ -0,0 +1,294 @@
+/**
+ *  Vision In-Wall 2Relays Switch (Models: ZL7435xx-5)
+ *
+ *  Author:
+ *    Lan, Kuo Wei 
+ *
+ *  Product Link: 
+ *    http://www.visionsecurity.com.tw/index.php?option=product&lang=en&task=pageinfo&id=335&belongid=334&index=0
+*/
+metadata {
+	definition(
+		name: "Vision In-Wall 2Relays Switch", 
+		namespace: "SmartThings", 

```

Comment:
Don't forget to check if child devices are working properly after You change the namespace.

Reply:
I can't change this now,
  because there have bug when namespace set to the other name,
  only can set to "SmartThings", otherwise child devices will dead to create,
  I think this is bug in Groovy IDE?

Reply:
This issue already contact support to check how to fix, please wait result.

Reply:
Already fix on 2021/02/22.